src/Adapters/MockAdapter.ts

```diff
@@ -1045,4 +1048,27 @@ export default class MockAdapter
             });
         }
     }
+
+    async getTimeSeriesData(_query: string) {
+        try {
+            await this.mockNetwork();
+
+            const mockData: Array<ADXTimeSeries> = [
+                {
+                    id: 'PasteurizationMachine_A01',
+                    key: 'InFlow',
+                    data: getMockTimeSeriesDataArrayInLocalTime(1)[0]
```

Comment:
In the past I have seen issues with snapshot type tests when timestamps are involved. Will this change what the chart looks like based on which machine it runs on?

Reply:
yeah this method returns data with datapoints with timetampt Data.now-somecertaintimeago. So the snapshots will change. okay I can send a flag to set the date from a certain date rather than Date.now which would produce new snapshot everytime.

Reply:
it only matters if it's visible in the chromatic test.

Reply:
in fact after thinking about it more, the solution I proposed ^ is not solving the snapshow issue. Because the method above is only for generating mock data points with timestamps, even if I control the data points timestamps, I draw the x axis of the chart again based on Date.now-quicktimespaninmillis. which is always going to be different in snapshots. So I am not going to make any changes in this method since it is not a full solution. Will think about it more and will keep this open.

Reply:
does the Axis need to use the current time? Can't it take in a time range? or is that part of the change that we asked for to have the axis always be for a given horizon even when there is no data?

Reply:
x axis min and max values are set by like this as we discussed in previous meetings: 
max is the right hand side of x axis and it is always now in local time
min is the left most side of x axis and it is always now-quicktimespaninmillis.

This is what user needs to see in data history widget based on the selected default quick time span in millis.

Reply:
yeah... makes sense. ok. hmm tricky one. I guess we could add some code in there for the isChromatic and then replace in a specific value or something?